{"name": "test_on_first", "status": "broken", "statusDetails": {"message": "TypeError: BrowserType.launch() got an unexpected keyword argument 'logger'", "trace": "self = <test_oncall.TestOnCallTestKeywords object at 0x0000015270BB3250>\n\n    @pytest.fixture()\n    def browser_setup(self):\n>       self.browser_on_call=self.OnCallFunctions.create_browser()\n\ntests\\OnCall\\test_oncall.py:19: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <controller.OnCallFunctions.OnCallFunctions object at 0x00000152708CBCB0>\n\n        def create_browser(self):\n            self.start()\n>           browser = self.pw.chromium.launch(headless=False,logger={\n            \"isEnabled\": lambda name, severity: name == \"api\",\n            \"log\": lambda name, severity, message, args: print(f\"{name}: {message}\")\n        }\n    )\nE   TypeError: BrowserType.launch() got an unexpected keyword argument 'logger'\n\ncontroller\\base.py:19: TypeError"}, "start": 1748332746285, "stop": 1748332746285, "uuid": "88780547-b0b3-4925-8556-1b7e73cf9422", "historyId": "e90402f55abcf97949df1b9058e18f85", "testCaseId": "e90402f55abcf97949df1b9058e18f85", "fullName": "tests.OnCall.test_oncall.TestOnCallTestKeywords#test_on_first", "labels": [{"name": "story", "value": "Smoke"}, {"name": "tag", "value": "critical"}, {"name": "tag", "value": "First"}, {"name": "feature", "value": "OnCall"}, {"name": "parentSuite", "value": "tests.OnCall"}, {"name": "suite", "value": "test_oncall"}, {"name": "subSuite", "value": "TestOnCallTestKeywords"}, {"name": "host", "value": "DESKTOP-G02SEED"}, {"name": "thread", "value": "24136-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.OnCall.test_oncall"}]}